doc/development/migration_guides.rst

```diff
@@ -24,4 +24,8 @@ each ``sf`` (v0.19.0) command:
    * - ``sf --ping``
      - ``xcc ping``
    * - ``sf run "foo.xbb"``
-     - ``xcc job submit --name "bar" --target "X8_01" --circuit "$(cat foo.xbb)"``
\ No newline at end of file
+     - ``xcc job submit --name "bar" --target "X8_01" --circuit "$(cat foo.xbb)"``
+
+.. note::
+
+  To submit a job using Windows PowerShell, simply replace ``cat foo.xbb`` with  ``Get-Content foo.xbb -Raw``.
```

Comment:
Does PowerShell also support the `$(...)` expansion syntax? I also wonder how many people tend to use PowerShell instead of cmd, or some other cmd-based alternative. 🤔

Reply:
> Does PowerShell also support the `$(...)` expansion syntax?

Yes! I'm not sure if the semantics are equivalent to Bash but it's close enough to make it work here.

> I also wonder how many people tend to use PowerShell instead of cmd, or some other cmd-based alternative. 🤔

My understanding is that Microsoft has preferred PowerShell over cmd for some time now so I think it's a safe bet to just show a PowerShell command as the Windows example.

Reply:
> My understanding is that Microsoft has preferred PowerShell over cmd for some time now

Good point, I'm just not so sure most Windows users actually use Powershell. 😆